src/config/configs/pagination.ts

```diff
@@ -0,0 +1,6 @@
+import { registerAs } from '@nestjs/config';
```

Comment:
in my opinion `src/config/configs/pagination.ts` is to complicated change it to flat structure `src/config/pagination.ts`